Oswaldo Vazquez Aparicio/26 Agosto/hackerrank/JavaOutputFormatting.java

```diff
@@ -0,0 +1,20 @@
+package hackerrank;
+
+import java.util.Scanner;
+
+public class JavaOutputFormatting {
+
+    public static void main(String[] args) {
+
+            Scanner sc=new Scanner(System.in);
+            String separatorLine = "================================"
+            System.out.println(separatorLine);
+            for(int i = 0;i < 3;i++){
```

Comment:
Even when all programming books teach us for loops with var names like i, x, y, z and so on, please avoid these.

https://javarevisited.blogspot.com/2014/10/10-java-best-practices-to-name-variables-methods-classes-packages.html#axzz74r1AYlmg